db/migrations/20220918104526-create-users-table.js

```diff
@@ -0,0 +1,40 @@
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('users', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
```

Comment:
in the ERD diagram I made email as the primary key since 
1. email is a bit more unique?
2. the issue with id is users with the same email can create multiple accounts (which doesn't seem right, unless in the controller code we check if there is an existing email with the user when he signs up)

Reply:
I actually did not know it was possible to use a field like email as a primary key - learnt sth new!
But it appears that the [consensus](https://stackoverflow.com/questions/3804108/use-email-address-as-primary-key#:~:text=Yes%2C%20it%20is%20a%20bad,to%20update%20their%20email%20addresses.&text=is%20not%20useful-,Show%20activity%20on%20this%20post.,use%20it%20as%20a%20key.) is that we ought not to use email as a primary key.
I did implement an email check in controller to deal with the multiple account problem that you raised, so hopefully that wouldn't be an issue.